docs/content/docs/install/settings.md

```diff
@@ -250,6 +250,16 @@ A few settings are available to configure this feature:
 
   Set this to the root URL of your custom console. example: `https://mycorp.com`
 
+* `custom-console-url-namespace`
+
+  Set this to the URL where to view the details of the `Namespace`.
+
+  The URL supports all the standard variables as exposed on the Pipelinerun (refer to
+  the documentation on [Authoring PipelineRuns](../authoringprs)) with the added
+  variable:
+
+  * `{{ namespace }}`: The target namespace where the pipelinerun is executed
+
```

Comment:
Can you add a example format how it looks like after adding {{ namespace }}

I mean something similar to what added in `custom-console-url-pr-details`
  example: `https://mycorp.com/ns/{{ namespace }}/pipelinerun/{{ pr }}`

Reply:
what do you mean for openshift? you only need this if you have a custom console and do not use the openshift console, you can have a custom console when running on openshift